quarkus-opentelemetry-exporter-sentry/runtime/src/main/java/io/quarkiverse/opentelemetry/exporter/sentry/config/SentryConfig.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkiverse.opentelemetry.exporter.sentry.config;
+
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+public class SentryConfig {
+
+    @ConfigMapping(prefix = "quarkus.opentelemetry.tracer.exporter.sentry")
```

Comment:
Please rename to `quarkus.otel.sentry`.

This to align with the new namespace conventions of the `quarkus-opentelementry` extension